webviz_config/templates/copy_data_template.py.jinja2

```diff
@@ -41,6 +42,13 @@ app._deprecated_webviz_settings = {
     "theme" : webviz_settings.theme
 }
 
+{% if logging_config_dict is defined %}
+# Apply a logging config dict as specified via the --logconfig command line argument. 
+logging_config_dict = {{ logging_config_dict }}
+print("Configuring logging from dictionary:", logging_config_dict)
```

Comment:
Do we want this to be printed, or is it remains after the development?

Reply:
I think maybe we should remove it here in the 'copy' template, but I'm inclined to leave it in place in the 'main' template so that when running a portable build it will apear in the console that a logging dictionary is being applied.